theme/templates/resource-landing-page/modals.html

```diff
@@ -1946,6 +1946,15 @@ <h3 class="panel-title">Sharing status</h3>
             </p>
         </div>
     {% endblock %}
+
+    {% block change-share-permission %}
+        <div id="dialog-confirm-change-share-permission" title="Warning! Loss of owner privilege" style="display:none">
+            <p><span class="ui-icon ui-icon-alert" style="float:left; margin:12px 12px 50px 0;"></span>
+                This action will make you lose owner privilege to this resource. You will no longer be an owner
```

Comment:
Update this text as recommended here: https://github.com/hydroshare/hydroshare/issues/2127#issuecomment-459900604

Reply:
Will do.